apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -87,18 +109,11 @@ export function HeaderNav({ isIntercomOpened }: Props) {
     }
   }, [boot, currentUser, currentOrganization]);
 
-  let themeTitle = 'Match System Appearance';
-  if (themeStatus === 'dark') {
-    themeTitle = 'Dark Theme';
-  } else if (themeStatus === 'light') {
-    themeTitle = 'Light Theme';
-  }
-
   const additionalMenuItems = useMemo(() => {
     return [
       {
         id: 'toggle-theme',
-        title: themeTitle,
+        title: getThemeTitleByColorMode(themeStatus),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** This will be called every render despite the color mode not changing -- perhaps it could be memoized ‏if we are worried about performance.